package.json

```diff
@@ -79,7 +79,8 @@
     "postversion": "node scripts/manage-version-suffix.mjs && git commit --no-verify -a -m \"[RELEASE] prepare version for new developments\"",
     "postinstall": "husky install",
```

Comment:
⚠️ the postinstall script should be removed!

Reply:
Done